compiler/apps/playground/components/Editor/Input.tsx

```diff
@@ -23,6 +23,15 @@ import React$Types from '../../node_modules/@types/react/index.d.ts';
 
 loader.config({monaco});
 
+declare global {
+  interface Window {
+    /**
+     * Set to true once the Monaco editor instance has mounted. Used as an imperative signal for any code that needs to defer until Monaco is available.
+     */
+    __MONACO_LOADED__?: true;
+  }
+}
```

Comment:
just curious, why did you need to add this?

Reply:
This resolves some Typescript errors by letting Typescript know about this global variable the Monaco editor sets. I'll remove since it's unrelated to this PR.